### PR TITLE
Support cluster connections to RabbitMQ.

### PR DESCRIPTION
**Pull request by calavera**
Shuffle the list of connections and try until it connects to one of them.

### CODE REVIEW

File: rabbitmq.go [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
 
 // RabbitConfig defines all that is necessary to connect to a
 type RabbitConfig struct {
 	tlsDefinition
-	URL string `json:"url"`
+	URL     string   `json:"url,omitempty"`
+	Cluster []string `json:"cluster,omitempty"`
```
Comment (rybit) [inline rabbitmq.go]: I think that we should support only cluster. To be more in line with the nats config 

```
// NatsConfig represents the minimum entries that are needed to connect to Nats over TLS
type NatsConfig struct {
  tlsDefinition

  Servers []string `json:"servers"`
}```

Comment (calavera) [inline rabbitmq.go]: Sounds good.

**Comment by rybit** [comment]
lg2m

**Comment by calavera** [comment]
@rybit I changed it to only have a slice of servers. Take a look.

**Comment by rybit** [comment]
shipit

**Pull request merged**